### PR TITLE
FreeBSD init: use cloudinit_enable as only rcvar

### PR DESCRIPTION
**Pull request by igalic**
## Proposed Commit Message

```
FreeBSD init: use cloudinit_enable as only rcvar

all components of cloudinit need to run, and in a specific order.
If cloudinit is to be enabled, it should only rely on one variable.
This change better encodes that, than #161 

Sponsored by: FreeBSD Foundation
```

## Additional Context
Currently, when cloud-init isn't enabled, some of its components issue rc warnings.
Previously, in https://github.com/canonical/cloud-init/pull/161 we went overboard, by giving each its own variable

## Test Steps
install this patch, reboot, see no more warnings.

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/contributing.html)
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Comment by igalic** [comment]
cc @Jehops

**Review by goneri — APPROVED** [review]

**Comment by Jehops** [comment]
We don't want `load_rc_config $rcvar`.  This will look to source files from places like `/etc/rc.conf.d/cloudinit_enable/*`.

**Comment by igalic** [comment]
> We don't want `load_rc_config $rcvar`. This will look to source files from places like `/etc/rc.conf.d/cloudinit_enable/*`.

oh, right, so, `load_rc_config $name` or `load_rc_config cloudinit` would be better here

**Comment by igalic** [comment]
updated the PR now.

**Comment by TheRealFalcon** [comment]
LGTM, but I'll wait for an approval from @Jehops since BSD isn't my wheelhouse.

**Comment by Jehops** [comment]
LGTM.  Thanks @igalic.

**Review by TheRealFalcon — APPROVED** [review]
Thanks!

**Pull request merged**
